src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/InfrastructureFormula.kt

```diff
@@ -59,16 +73,17 @@ class InfrastructureFormula<out T : VirtualUsers>(
         }
 
         val network = networkProvisioning.get()
-        val provisionJira = executor.submitWithLogContext("jira") {
-            overrideJiraNetwork(network).provision(
+
+        val provisionedJira = if (userProvisionedJira != null) userProvisionedJira else executor.submitWithLogContext("jira") {
```

Comment:
this can be simplified as
`val provisionedJira = userProvisionedJira ?: executor.submitWithLogContext("jira")`

Reply:
Yeahhh- Elvis :)